webpack.config.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
```

Comment:
- [OPTIONAL] `Linters` are tools that perform static analysis on software in order to recognize and report adherence/non-adherence to some set of coding best practices. Instead of disabling `eslint`, it will be nice to fix the issue.